foreshadow/steps/feature_summarizer.py

```diff
@@ -1,21 +1,27 @@
 # noqa
 import json
 
-from foreshadow.intents import Droppable
-from foreshadow.utils import AcceptedKey, get_transformer
+import pandas as pd
+
+from foreshadow.utils import AcceptedKey, get_transformer, standard_col_summary
 
 from .preparerstep import PreparerStep
 
 
 class FeatureSummarizerMapper(PreparerStep):  # noqa
-    def __init__(self, **kwargs):
+    def __init__(self, y_var=False, problem_type=None, **kwargs):
```

Comment:
Can we rename `y_var` to `is_y_var` since it is a boolean variable?

Reply:
I did some search and found 100+ matches for y_var. I suggest we have a dedicated PR to change this.